src/mktemp.js

```diff
@@ -0,0 +1,88 @@
+var path = require('path');
+var fs = require('fs');
+var common = require('./common');
+var mkdir = require('./mkdir');
+var touch = require('./touch');
+var rm = require('./rm');
+var chmod = require('./chmod');
+
+var tempdir;
+var DEFAULT_TEMPLATE = 'tmp.shelljs.XXXXXXXXXXXXXXXXXXXX';
```

Comment:
We should probably just rely on the value of `tempdir`, since that's cross-platform